docs/src/model_setup/forcing_functions.md

```diff
@@ -277,3 +280,88 @@ ContinuousForcing{Nothing} at (Center, Center, Center)
 └── field dependencies: (:T,)
 ```
 
+## `AdvectiveForcing`
+
+`AdvectiveForcing` defines a forcing function that represents advection by
+a separate or "slip" velocity relative to the prognostic model velocity field.
+`AdvectiveForcing` is implemented with native Oceananigans advection operators,
+which means that tracers advected by the "flux form" advection term
+``∇ ⋅ u⃗ˢˡⁱᵖ c``. Caution is advised when ``u⃗ˢˡⁱᵖ`` is not divergence free.
```

Comment:
I don't know if it is something with my browser, but the unicode characters look strange here. Is this suppose to look like this from here, but will be fine on the documentation?

Reply:
They look a little strange to me too unfortunately.